core/src/main/java/org/opentosca/toscana/core/api/CommonController.java

```diff
@@ -16,56 +10,23 @@
 import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
 
 /**
- This REST Controller handles the requests which do not relate to platforms or csars.
- <p>
- Currently the only request handled by this controller is the {@code status} request
- <p>
- For sample Responses of the Requests, please have a look at docs/api/api_samples.md
+ * This REST Controller handles the requests which do not relate to platforms or csars. <p> Currently the only request
```

Comment:
Please reimport the codestyle as described [here](https://github.com/StuPro-TOSCAna/TOSCAna/issues/76#issuecomment-340316761) and perform a reformat.